spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StdCmdConfig.scala

```diff
@@ -246,6 +246,16 @@ object StdCmdConfig {
             failure("The --is-xcom option is supported only for COBOL data format")
           })
 
+      opt[Boolean]("cobol-is-text").optional().action((value, config) => {
+        config.copy(cobolOptions = cobolSetIsText(config.cobolOptions, value))
+      }).text("Specifies if the mainframe file is ASCII text file")
+        .validate(value =>
+          if (rawFormat.isDefined && rawFormat.get.equalsIgnoreCase("cobol")) {
```

Comment:
Maybe `rawFormat.get` could cause NullPointerException

Reply:
It won't since it is validated earlier. The `scopt` way of things is a big ugly, I agree. I'm just following how we handle other cmd parameters. Maybe we can refactor StdCmdConfig in the future.